Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'simplecov', '>= 0.19'
 gem 'yard'
 
 # TODO: Move to gemspec when RuboCop RSpec v3 is released.
-gem 'rubocop-rspec', '~> 2.27'
+gem 'rubocop-rspec', '>= 3.0.0.pre'
```

Comment:
@ydah Should I move this restriction into the gemspec, and then we do a prerelease of rubocop-rspec_rails too? Or should we just wait for rubocop-rspec v3.0.0 proper?